docs/changelog.md

```diff
@@ -139,6 +139,11 @@ what we publish.
 
 - Added `StringSlice(..)` initializer from a `StringLiteral`.
 
+- Added a `byte_length()` method to `String`, `StringSlice`, and `StringLiteral`
```

Comment:
In the PR title and description you mention some changes to `String.split` I don't see them in this patch. Can you please update it, or describe them here?

Reply:
It is actually just a small fix adding `sep: NoneType = None` as default for `String.split()`